ci_framework/roles/install_yamls/tasks/main.yml

```diff
@@ -14,6 +14,12 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+
+- name: "Get environment structure"
+  register: get_makefile_env_output
+  get_makefile_env:
+    makefile_path: "{{ cifmw_install_yamls_repo }}/Makefile"
```

Comment:
You'd be missing the devsetup/Makefile content.

Reply:
We'd need to consider the "cost" of calling the module twice. Proposal: what about "find all Makefile in tree" and iterate on them all?
The idea was mostly to avoid calling 2 times the same action, and I think we should stick to it. Plus, if we use a kind of "find", we'd be able to cover future changes to the Makefile locations... ?

Reply:
Sounds good to me. I'll change the behavior